cmd/root.go

```diff
@@ -5,14 +5,43 @@
 package cmd
 
 import (
+	"log"
+
+	"github.com/elastic/elastic-agent-changelog-tool/internal/settings"
 	"github.com/spf13/cobra"
 )
 
 const defaultOwner = "elastic"
 const defaultRepo = "elastic-agent"
 
+var config settings.Config
+
+func GetOwner(flagOwner string) string {
+	switch {
+	case config.Owner != "":
+		return config.Owner
+	default:
+		return flagOwner
+	}
+}
+
+func GetRepo(flagRepo string) string {
+	switch {
+	case config.Repo != "":
+		return config.Repo
+	default:
+		return flagRepo
+	}
+}
+
 // RootCmd creates and returns root cmd for elastic-agent-changelog-tool.
 func RootCmd() *cobra.Command {
+	var err error
+
+	config, err = settings.LoadConfig()
```

Comment:
Move config loading to `main.go` file, and use `log.Fatal` when file cannot be parsed, as this is an unrecoverable error (we cannot assume what configuration parameters have been changed and this can seriously impact the execution)